config/initializers/commontator.rb

```diff
@@ -63,9 +64,7 @@
   # Default: lambda { |user, view|
   #            view.commontator_gravatar_image_tag(
   #              user, 1, :s => 60, :d => 'mm') }
-  config.user_avatar_proc = lambda { |user, view|
-                                     view.commontator_gravatar_image_tag(
-                                       user, 1, :s => 60, :d => 'mm') }
+  config.user_avatar_proc = lambda { |user, view| }
```

Comment:
Just blank avatars, plus Gravatar enables tracking. Plus I'm not sure I want to encourage the kind of images that students tend to select…